PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalRecurringBillingDetails.kt

```diff
@@ -22,6 +22,8 @@ import org.json.JSONObject
  * @property productQuantity Quantity associated with the product.
  * @property shippingAmount The shipping amount for the billing cycle at the time of checkout.
  * @property taxAmount The taxes for the billing cycle at the time of checkout.
+ * @property unitAmount This property is for checkout with vault requests only.
+ * @property amountBreakdown Breakdown of items associated to the total cost
  */
```

Comment:
Both of these properties should specify that they are for checkout with vault requests only, but they should also include a description of what the property is first.

Ex: `@property amountBreakdown Breakdown of items associated to the total cost - this property is for checkout with vault requests only`